packages/component/src/Attachment/TextContent.js

```diff
@@ -13,10 +17,13 @@ import useStyleSet from '../hooks/useStyleSet';
 const TextContent = ({ contentType, text }) => {
   const renderMarkdownAsHTML = useRenderMarkdownAsHTML();
   const [{ textContent: textContentStyleSet }] = useStyleSet();
+  const strippedText = remark()
```

Comment:
'useMemo'?

Reply:
This is making use of the packages remark and stripMarkdown -- just following the recommended syntax from there.